scss/_settings_themes.scss

```diff
@@ -8,3 +8,4 @@ $theme-default-p-contextual-menu: 'light' !default;
 $theme-default-p-inline-list--middot: 'light' !default;
 $theme-default-p-button: 'light' !default;
 $theme-default-p-chip: 'light' !default;
+$theme-default-p-tabs: 'light' !default;
```

Comment:
I know we do it for other component, but it's probably the most useless part of theming in Vanilla. Will we ever wanna change the default? Would we expect anyone to change it?

I don't want to revisit it now for existing components, but if we can avoid adding new unnecessary settings I guess it's a good start.

Reply:
Ok. Let's keep it consistent and leave the discussions for later.